content/03-reference/01-tools-and-interfaces/07-preview-features/001-mssql/001-mssql-start-from-scratch.mdx

```diff
@@ -233,13 +235,13 @@ To create tables using the `sqlcmd`:
 1. Use the `sqlcmd` tool to run the `create.sql` script, replacing `mypassword` with your password:
 
   ```terminal
-  /opt/mssql-tools/bin/sqlcmd -S localhost -U SA -P "mypassword" -i create.sql
```

Comment:
mm, these paths all should actually not be changed!

Reply:
Find-and-replace giveth and it taketh away.